e2e/__tests__/components/twitter-feed-test.js

```diff
@@ -2,7 +2,7 @@ const puppeteer = require('puppeteer');
 const ct = require('../../config/constants');
 let browser;
 let page;
-
+jest.setTimeout(40000);
```

Comment:
what was the original timeout time?

Reply:
the default is 30000 mx, this test imtermittenly run for 37s and fail